src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        core.basesyntax.db.Storage.people.clear();
+    }
+
+    @Test
+    public void register_validUser_userRegistered() {
+        User user = new User();
+        user.setLogin("validLogin");
+        user.setPassword("validPass");
+        user.setAge(20);
+
+        User registeredUser = registrationService.register(user);
+
+        assertNotNull(registeredUser);
+        assertEquals(user.getLogin(), registeredUser.getLogin());
+    }
+
+    @Test
+    public void register_existingLogin_userNotRegistered() {
+        User user1 = new User();
+        user1.setLogin("existingLogin");
+        user1.setPassword("password1");
+        user1.setAge(20);
+        registrationService.register(user1);
```

Comment:
```suggestion
        Storage.people.add(user);
```

read common mistakes file